app/ui-flextab/client/tabs/uploadedFilesList.js

```diff
@@ -32,7 +33,8 @@ Template.uploadedFilesList.helpers({
 	},
 
 	url() {
-		return getURL(`/file-upload/${ this._id }/${ this.name }`);
+		const fileNameConversion = path.normalize(this.name).replace(/^(\.\.(\/|\\|$))+/, ' ');
+		return `/file-upload/${ this._id }/${ encodeURI(fileNameConversion) }`;
```

Comment:
Please do not remove the `getUrl` function here, because it is used in case the server is running in a subdir.

Reply:
Sure, let me try and fix it